### PR TITLE
[airq] Improve error handling

### PR DESCRIPTION
**Pull request by fwolter**
The current implementation shows "probably wrong pasword" on most errors, but not all of them are related to a wrong password. E.g. a network error or failing of the decryption caused by limited JVM crypto strength, due to US import restrictions 🙈.

Also, some null warnings were fixed, JSON syntax errors are handled, and network operations in `initialize()` were removed.

And when the connection failed, the ThingStatus was flapping between ONLINE and OFFLINE, although a connection has never been established.

### CODE REVIEW

File: bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java [hunk 1]
```diff
@@ -460,11 +427,14 @@ public int getResponseCode() {
 
     @Override
     public void dispose() {
-        if (pollingJob != null) {
-            pollingJob.cancel(true);
+        ScheduledFuture<?> localPollingJob = pollingJob;
+        if (localPollingJob != null) {
+            localPollingJob.cancel(true);
```
Comment (lsiepel) [inline bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java:432]: Don't you also need to set pollingJob = null same for getConfigDataJob; down below?

Comment (fwolter) [reply bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java:432]: It won't harm, but it also has no benefit, or do I miss something? I saw this pattern in other bindings, but I never understood the purpose...

Comment (lsiepel) [reply bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java:432]: Same here, so we have the same questions :-)

**Review by lsiepel — COMMENTED** [review]
Thanks, LGTM

**Comment by fwolter** [comment]
Thanks!

**Pull request merged**